examples/index.js

```diff
@@ -26,18 +26,25 @@ const hawkularAPM = require('../index');
 const SERVER_PORT = 9000;
 
 opentracing.initGlobalTracer(new hawkularAPM.APMTracer({
-    recorder: new hawkularAPM.ConsoleRecorder(),
+    recorder: new hawkularAPM.HttpRecorder('http://localhost:9411', 'jdoe', 'password'),
+    // recorder: new hawkularAPM.ConsoleRecorder(),
     sampler: new hawkularAPM.AlwaysSampledSampler(),
 }));
 
 // /hello handler
 dispatcher.onGet('/hello', function(req, res) {
     const serverSpan = opentracing.globalTracer().startSpan('hello', {
+        childOf: extractSpanContext(opentracing.globalTracer(), req.headers),
         tags: {
+            'http.method': 'GET',
             'http.url': extractUrl(req),
             'http.status_code': 200,
```

Comment:
Could you set this tag separately nearer the `finish` method, as usually the status would only be known at the end of handling a request.